RadixWallet/Clients/AppPreferencesClient/AppPreferencesClient+Live.swift

```diff
@@ -35,6 +33,15 @@ extension AppPreferencesClient: DependencyKey {
 					profile.id,
 					isEnabled ? .enable : .disable
 				)
+			},
+			setIsCloudBackupEnabled: { isEnabled in
```

Comment:
When would we use `setIsCloudBackupEnabled` and not `setIsCloudProfileSyncEnabled`?

Reply:
But as of this PR, it is still in use, so it shouldn't be deleted yet